Utils/Selectors/pageSels.js

```diff
@@ -4,7 +4,22 @@
 export default {
     homePage: {
         signInBtn: 'div[data-testid="account"]',
-        signIn: 'Sign In'
+        signIn: 'Sign In',
+        menuBtn: 'a[data-testid="navItems-component-Menu"]',
+        menuSearchBar: 'input[data-testid="search-our-menu"]',
+        searchResultTitle: 'h1[class*="kfc-menu-title"]',
+        addToCartText: 'Add to cart',
+        itemCard: 'div[data-testid=menu-card-content-pdp-nav] ',
+        itemNameInCard: 'div[class="medium-menu-product-header"]',
+        deliveryTypeBtn: 'button[data-testid="disposition-order-click-handler-Disposition - Delivery"]',
+        pickUpTypeBtn: 'button[data-testid="disposition-order-click-handler-Disposition - Pickup]',
+        store_area_street_searchBar: 'input[data-testid="store-search-input"]',
+        store_area_street_suggestionList: 'div[class="suggestion-list"]',
```

Comment:
Have one specific format, snake or Camel.

best way is the have camel casing

Reply:
Okay, I will update that.